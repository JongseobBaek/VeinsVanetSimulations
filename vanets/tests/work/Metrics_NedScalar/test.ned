package vanets.tests.work.Metrics_NedScalar;

simple Test
{
	@isNetwork(true);
}

