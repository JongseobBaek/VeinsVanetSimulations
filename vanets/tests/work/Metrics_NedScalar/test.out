OMNeT++ Discrete Event Simulation  (C) 1992-2014 Andras Varga, OpenSim Ltd.
Version: 4.6, build: 141202-f785492, edition: Academic Public License -- NOT FOR COMMERCIAL USE
See the license for distribution terms and warranty disclaimer
Setting up Cmdenv...
Loading NED files from /home/srodriguez/Documents/Research/VeinsVanetSimulationTest/vanets/tests/work/Metrics_NedScalar: 2
Loading NED files from /home/srodriguez/Documents/Research/VeinsVanetSimulationTest/vanets/src: 7
Loading NED files from /home/srodriguez/Documents/Research/VeinsVanetSimulationTest/inet/src: 274
Loading NED files from /home/srodriguez/Documents/Research/VeinsVanetSimulationTest/veins/src/veins: 30

Preparing for running configuration General, run #0...
Scenario: $repetition=0
Assigned runID=General-0-20160808-14:05:49-8827
Setting up network `vanets.src.RSUScenario'...
Initializing...
0.524407
0
0.522442
0

Running simulation...
** Event #1   T=0   Elapsed: 0.000s (0m 00s)  0% completed
     Speed:     ev/sec=0   simsec/sec=0   ev/simsec=0
     Messages:  created: 13   present: 13   in FES: 4
0.418794
0.1
0.4388325558
3.1
0.9127983166
6.1
0.3341207687
9.1
0.9393091714
12.1
0.6394397668
15.1
0.9223344593
18.1
0.1431661723
21.1
0.1415956817334518
24.1
0.3349040479
27.1
0.7018686404
30.1
0.5380786679
33.1
0.8457772154
36.1
0.6497396387
39.1
** Event #975   T=40   Elapsed: 1.332s (0m 01s)  100% completed
     Speed:     ev/sec=731.231   simsec/sec=30.03   ev/simsec=24.35
     Messages:  created: 476   present: 73   in FES: 5

<!> Simulation time limit reached -- simulation stopped at event #975, t=40.


Calling finish() at end of Run #0...
Writing TraCI message of 2 bytes
Reading TraCI message of 7 bytes

End.
